Source/Microsoft.Teams.Apps.ExpertFinder/Bots/ExpertFinderBot.cs

```diff
@@ -586,17 +516,9 @@ private async Task<MessagingExtensionResult> GetSearchResultAsync(string searchQ
         /// <returns>A task that represents typing indicator activity.</returns>
         private async Task SendTypingIndicatorAsync(ITurnContext turnContext)
         {
-            try
-            {
-                var typingActivity = turnContext.Activity.CreateReply();
-                typingActivity.Type = ActivityTypes.Typing;
-                await turnContext.SendActivityAsync(typingActivity);
-            }
-            catch (Exception ex)
-            {
-                // Do not fail on errors sending the typing indicator
-                this.logger.LogWarning(ex, $"Failed to send a typing indicator: {ex.Message}");
-            }
```

Comment:
is it fixed now for EU region?

Reply:
I didn't mean to remove this-- it wasn't added it in the internal repo though.